pysumma/Simulation.py

```diff
@@ -65,20 +70,21 @@ def execute(self, run_suffix, run_option):
             print(cmd)
         else:
             raise ValueError('No executable defined. Set as "executable" attribute of Simulation or check run_option')
-
-        # run shell script in python and print(shlex.split(cmd))
-        cmd = shlex.split(cmd)
-        p = subprocess.Popen(cmd, stdout=subprocess.PIPE)
-        output = p.communicate()[0].decode('utf-8')
-        print(output)
-        if 'FATAL ERROR' in output:
-            raise Exception("SUMMA failed to execute!")
-
+        # run shell script in python
+        preprocess = []
+        if self.library_path:
+            preprocess = ['export LD_LIBRARY_PATH="{}" && '.format(self.library_path)]
+        if arglist:
+            preprocess.append('{} && '.join(arglist))
+        preprocess = "".join(preprocess)
+        cmd = preprocess + " && " + cmd
+        proc = subprocess.Popen(cmd, shell=True, stdin=subprocess.PIPE,
+                stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         # define output file name
         out_file_path = self.output_path.filepath + \
-                        self.output_prefix.value + '_output_' + \
+                        self.output_prefix.value + 'output_' + \
```

Comment:
I think we have to edit SUMMA Test Case before merging this commit. Do you have updated Test Cases and SUMMA version?